CHANGELOG.md

```diff
@@ -6,8 +6,11 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 5.4.0
+
 ### General
 
+* `FEAT`: support Camunda 8.1 and 7.18 as target platform ([#3158](https://github.com/camunda/camunda-modeler/issues/3158))
 * `DEPS`: bump @camunda/linting to v0.7.2 ([#3184](https://github.com/camunda/camunda-modeler/pull/3184))
```

Comment:
Shall we include the core dependency bumps (camunda-bpmn-js, bpmn-js, ...) here, too? Just a good practice I believe.

Reply:
added with 00352244cf30fa9f0afe8cfe5996b1cc7f5485df